lib/src/test/kotlin/com/swmansion/starknet/data/TransactionHashCalculatorTest.kt

```diff
@@ -61,7 +62,7 @@ internal class TransactionHashCalculatorTest {
 
     @Nested
     inner class TransactionHashV3Test {
-        private val chainId = StarknetChainId.GOERLI
+        private val chainId = StarknetChainId.SEPOLIA
```

Comment:
Nit: 

I would stick to using `SN_GOERLI` in these tests for now to avoid changes to hashes, similarly to how you handle `DeprecatedTransactionHashTest`